composer/spec/dependabot/composer/file_fetcher_spec.rb

```diff
@@ -59,6 +59,13 @@
       to match_array(%w(composer.json composer.lock))
   end
 
+  it "provides the composer version" do
+    expect(file_fetcher_instance.package_manager_version).to eq({
+      ecosystem: "composer",
+      package_managers: { "composer" => "1" }
```

Comment:
Unfortunately, both of these tests are against a fixture that was generated using `composer 1`.

I thought about fixing, but we'll need to update this fixture anyway when we drop support for `composer 1`, so no sense partially doing it here, only to completely refactor later.

We _do_ have some UT test coverage of `"2"` because the code that actually generates the version number is used to accessing the native helper... if it wasn't spitting out `"2"`, all those unit tests would be failing.